client/modules/wj-css/src/base/_base.scss

```diff
@@ -0,0 +1,55 @@
+// Normalize html/body
+html,
+body {
+  min-width: 100%;
+  max-width: 100%;
+  min-height: 100%;
+  font-size: 1rem;
+  line-height: 1.5;
+  touch-action: manipulation;
```

Comment:
Why `manipulation` (which according to MDN prevents double-tap-to-zoom) over `auto`?

Reply:
Double tapping to zoom introduces a 300ms latency for _all interaction taps_. If your UI doesn't require you to zoom (because it isn't a desktop site), you can just use pinch zooming for the scant times you actually need to zoom.